src/AltinnCore/Common/Models/DeploymentResponse.cs

```diff
@@ -0,0 +1,28 @@
+namespace Common.Models
```

Comment:
Don't think its necessary to have two models, all the fields can just be in one model

Reply:
Fixed by extending :)